docs/UserGuide.md

```diff
@@ -451,7 +464,16 @@ Exits focus mode.
 Format: `unfocus`
 * Alternatively, you can exit focus mode by clicking on the button shown [here](#enter-focus-mode-focus).
 
-## FAQ
+## Frequently Asked Questions (FAQ)
+
+**Q**: How do I install Java?
+
+**A**: You may refer to [Oracle's JDK Installation Guide](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html#GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A).
+
+**Q**: I double-clicked the JAR file but was unable to start the app. What do I do?
+
+**A**: Open your preferred terminal and navigate to the directory the JAR file is located in. Then type in `java -jar TaAssist.jar` to run the application.
```

Comment:
Hmm, I think this instruction could be challenging to write without adding a lot of extra text. Since we're focusing on CLI users, would it then be okay to assume they can use the terminal to at least a novice level?

Reply:
Actually, that makes sense too. Should be fine to leave it out I guess